telemetry/definitions/commonDefinitions.json

```diff
@@ -982,6 +982,21 @@
             "type": "string",
             "description": "The API request ID"
         },
+        {
+            "name": "codeTransformMavenCommands",
+            "type": "string",
+            "description": "Maven commands that have been run."
+        },
+        {
+            "name": "codeTransformMavenExecutionSuccessful",
+            "type": "boolean",
+            "description": "If the Maven commands executions are successful."
```

Comment:
There is a standard `result` field on every metric. That should strongly be preferred to represent success/fail/cancel

Reply:
As @dhasani23 suggested, this result is already captured by codeTransform_mvnBuildFailed, so I just removed this field.